core/src/main/java/io/grpc/internal/Stream.java

```diff
@@ -79,6 +79,9 @@
    */
   boolean isReady();
 
+
+  void start();
```

Comment:
Probably only do it for ClientStreams? There is no race on ServerStreams (or rather, it is already handled).